client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -83,14 +88,20 @@ function createQuestionEpic(action$, state$, { window }) {
         )}\n${i18next.t('forum-help.add-code-three')}\n\n\`\`\`\n${endingText}`
       );
 
+      const titleText = dedent(
+        `${i18next.t(`intro:${superBlock}.title`)} - ${challengeTitle} - ${
+          user.name || user.email
```

Comment:
Emails are PII and aren't public information on `/learn` - I'm not sure we should use them as a fallback here.

Reply:
Good call. Is there even any way the name can be unset? I get a "too short" error if I try to unset my name - so I would guess not. Then I can just remove the email as fallback.

Reply:
The name is empty initially, but we could fallback to `user.username`.

Reply:
Sounds good! I pushed an update for that.

Reply:
Thanks that makes sense.